Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -481,6 +481,11 @@
 #define ENABLE_POINTER_LOCK 0
 #endif
 
+
+#if !defined(ENABLE_POST_EDITING_GRAMMAR_CHECKING) && ((PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 130000) || (PLATFORM(IOS_FAMILY) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 160000))
```

Comment:
IOS_FAMILY && __IPHONE_OS_VERSION_MIN_REQUIRED can't be right because __IPHONE_OS_VERSION_MIN_REQUIRED is stuck for watch/tv/etc.